### PR TITLE
Leftover vertical org changes

### PR DESCRIPTION
**Pull request by vincanger**
- [x] created new `newsletter` dir
- [x] `user.sendEmail` -> `user.sendNewsletter`
- [x] organized other leftover client and server files
- [x] update app_diff
- [x] update docs

### CODE REVIEW

File: opensaas-sh/blog/src/content/docs/start/guided-tour.md [hunk 1]
```diff
@@ -47,10 +47,14 @@ At the root of our project, you will see three folders:
 
 `e2e-tests` contains the end-to-end tests using Playwright, which you can run to test your app's functionality.
 
+### App File Structure
+
+We've structured this full-stack app template according to feature. That means that most directories within `app/src` contain both the React client code and NodeJS server code necessary for implementing its logic.
```
Comment (Martinsos) [inline opensaas-sh/blog/src/content/docs/start/guided-tour.md:52]:
```suggestion
We've structured this full-stack app template vertically (by feature). That means that most directories within `app/src` contain both the React client code and NodeJS server code necessary for implementing its logic. 
```

File: opensaas-sh/blog/src/content/docs/start/guided-tour.md [hunk 2]
```diff
@@ -106,38 +107,6 @@ Wasp abstracts away some things that you would normally be used to doing during
 It's possible to learn Wasp's feature set simply through using this template, but if you find yourself unsure how to implement a Wasp-specific feature and/or just want to learn more, a great starting point is the intro tutorial in the [Wasp docs](https://wasp-lang.dev/docs) which takes ~20 minutes.
 :::
 
-### Client
```
Comment (Martinsos) [inline opensaas-sh/blog/src/content/docs/start/guided-tour.md:109]: what about this stuff, isnt there some stuff still left in clinet and server that we want to explain here?

Comment (vincanger) [reply opensaas-sh/blog/src/content/docs/start/guided-tour.md:109]: `├── client/            # Shared components, hooks, landing page, and other client code (React).`
`├── server/            # Scripts, shared server utils, and other server-specific code (NodeJS).`

Comment (vincanger) [reply opensaas-sh/blog/src/content/docs/start/guided-tour.md:109]: Ok I added those sections back and revised them a bit :)

**Pull request merged**